src/modules/ScrollBar/index.js

```diff
@@ -0,0 +1,26 @@
+import SimpleBar from 'simplebar'
+import 'simplebar/dist/simplebar.css'
+
+export default class ScrollBar {
+    constructor (selector, options = {}) {
+        this.selector = selector
+        this.options = options
+        this.init()
+    }
+
+    init () {
+        this.createScrollBar()
+    }
+
+    createScrollBar () {
```

Comment:
This loop should be managed by a manager instead of a clase, 1 instance per class